src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -242,105 +244,279 @@
 							BorderBrush="{ThemeResource ControlStrokeColorDefault}"
 							BorderThickness="0,0,0,1"
 							PointerPressed="Grid_PointerPressed">
+
+							<!--  Header Grid MenuFlyout  -->
 							<Grid.ContextFlyout>
 								<MenuFlyout>
-									<MenuFlyoutItem Click="SizeAllColumnsToFit_Click" Text="{helpers:ResourceString Name=SizeAllColumnsToFit}" />
+
+									<!--  All Columns To Fit  -->
+									<MenuFlyoutItem
+										x:Name="AllColumnsToFitMenuflyoutItem"
+										Click="SizeAllColumnsToFit_Click"
+										Text="{helpers:ResourceString Name=SizeAllColumnsToFit}" />
+
+									<!--  (Divider)  -->
 									<MenuFlyoutSeparator />
+
+									<!--  Git Unmerged File Kind  -->
+									<ToggleMenuFlyoutItem
+										x:Name="ShowItemGitUnmergedFileKindColumnVisibilityToggleMenuFlyoutItem"
+										Click="ToggleMenuFlyoutItem_Click"
+										IsChecked="{x:Bind ColumnsViewModel.GitStatusColumn.UserCollapsed, Mode=TwoWay, Converter={StaticResource BoolNegationConverter}}"
+										Text="{helpers:ResourceString Name=GitStatus}"
+										Visibility="{x:Bind ColumnsViewModel.GitStatusColumn.IsHidden, Mode=OneWay, Converter={StaticResource NegatedBoolToVisibilityConverter}}" />
+
+									<!--  Git Last Commit Date  -->
+									<ToggleMenuFlyoutItem
+										x:Name="ShowItemGitLastDateCommitColumnVisibilityToggleMenuFlyoutItem"
+										Click="ToggleMenuFlyoutItem_Click"
+										IsChecked="{x:Bind ColumnsViewModel.GitLastCommitDateColumn.UserCollapsed, Mode=TwoWay, Converter={StaticResource BoolNegationConverter}}"
+										Text="{helpers:ResourceString Name=GitLastCommitDate}"
+										Visibility="{x:Bind ColumnsViewModel.GitLastCommitDateColumn.IsHidden, Mode=OneWay, Converter={StaticResource NegatedBoolToVisibilityConverter}}" />
+
+									<!--  Git Last Commit Message  -->
+									<ToggleMenuFlyoutItem
+										x:Name="ShowItemGitLastCommitMessageColumnVisibilityToggleMenuFlyoutItem"
```

Comment:
Can you shorten the name? This is pretty long...

Reply:
I guess this is the most formal. What would you suggest me to make it be?

Reply:
`ShowGitCommitColumnMenuFlyoutItem`

Reply:
Sure, but it'll be ShowGitCommitColumnToggleMenuFlyoutItem

Reply:
That works, can you do the same for the other ones?

Reply:
Let me know when this is done

Reply:
Done